core-backend/common/src/mock.rs

```diff
@@ -247,6 +248,13 @@ impl Ext for MockExt {
     fn runtime_cost(&self, _costs: RuntimeCosts) -> u64 {
         0
     }
+
+    fn pre_process_memory_accesses(
+        _reads: &[(u32, u32)],
+        _writes: &[(u32, u32)],
```

Comment:
Same thing about newtypes. I left comment on that in memory.rs

Reply:
Done